### PR TITLE
Fix package.json validation issues in stacke_app

### PR DESCRIPTION
**Pull request by azisnaufal**
Verified by package-json-validator.com .

### CODE REVIEW

File: stackle_app/package.json [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+ "repository": {
+    "type": "git",
+    "url": "git+https://github.com/scorelab/Stackle.git"
+  },
+  "keywords": [
+    "stackle",
+    "SCORE"
+  ],
+  "author": "SCoRe Lab",
+  "license": "ISC",
```
Comment (agentmilindu) [inline stackle_app/package.json]: @scorelab/stackle-maintainers What should be the licence?

**Comment by azisnaufal** [comment]
Hi!
I just refer the value from this file.
https://github.com/scorelab/Stackle/blob/master/stackle_api/package.json

I'm sorry if i do any mistake. Should i repair it?

**Comment by psnmissaka** [comment]
Sorry for the late reply. 
The license is [Apache 2.0](https://github.com/scorelab/Stackle/blob/master/LICENSE)

File: stackle_app/package.json (hunk 1 above)
Comment (tharindupr) [reply stackle_app/package.json]: Apache License Version 2.0

**Pull request merged**